project/src/main/java/com/google/sps/Models/Document.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.models;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class Document{
+  String language, name, hash;
+  ArrayList<Long> userIDs = new ArrayList<Long>();
+  
+  Document(String name, String language, String hash, ArrayList<Long> userIDs) {
```

Comment:
Did we decide to store a static version of the list of userIDs? Wouldn't it be better if we didn't store the list of userIDs and instead each time that we want to get the list we check the database so if users are added after the document object is created the change can be reflected?

Reply:
I thought about it and for where we are going I believe a static version would suit our needs better, as when you share it with someone their ID gets appended to the array. Either way works though honestly.

Reply:
i think the current version is fine, since it is more time-efficient to append when users are added vs check if there are new users later

Reply:
maybe add a comment in the code that summarizes what you were saying above

Reply:
Added comment.